client/visualizer/package.json

```diff
@@ -57,7 +58,7 @@
     "productName": "Battlecode Client",
     "compression": "maximum",
     "files": [
-      "bc20/*",
+      "bc20/**/*",
```

Comment:
hmm

Reply:
The CopyWebpackPlugin now copies the speedscope code to "bc20/speedscope". "bc20/*" non-recursively takes all contents inside the "bc20/" directory and skips the files inside "bc20/speedscope/" because that's a different directory. I might be wrong here though.